cmd/harmony/main.go

```diff
@@ -102,6 +102,8 @@ var (
 	shardID            = flag.Int("shard_id", -1, "the shard ID of this node")
 	enableMemProfiling = flag.Bool("enableMemProfiling", false, "Enable memsize logging.")
 	enableGC           = flag.Bool("enableGC", true, "Enable calling garbage collector manually .")
+	blsKeyFile         = flag.String("blskey_file", "", "The encrypted file of bls serialized private key by passphrase.")
+	blsPassFile        = flag.String("blspass_file", "", "The file containing passphrase to decrypt the encrypted bls file.")
```

Comment:
Please use the `utils.GetPassphraseFromSource`.  Examples can be found in 493c3528f421b7cc19fcc6e60fd94aaaee668d5a.

Reply:
For consistency with the account key side, the flag should be named `-blspass`:
```suggestion
	blsPass            = flag.String("blspass", "", "how to get passphrase for the BLS key")
```